sys/sys/elf_common.h

```diff
@@ -803,6 +803,7 @@ typedef struct {
 #define	NT_FREEBSD_NOINIT_TAG	2
 #define	NT_FREEBSD_ARCH_TAG	3
 #define	NT_FREEBSD_FEATURE_CTL	4
+#define	NT_CHERIBSD_ABI_TAG		5
```

Comment:
The vendor is different so this can start from 1 again in its own namespace and avoid conflicts when upstream adds new ones

Reply:
I was worried that some 3rd-party software could use the last tag value instead of a pair (tag, vendor) that is recognised by it. If it's the case, it would fail with the unrecognised CheriBSD vendor name.

Reply:
Then that code is totally broken. You can't even decode the type without knowing the vendor, it can be (but normally isn't) something other than a 4-byte integer.

Reply:
(and you'd already get issues with FreeBSD notes colliding with those for NetBSD, OpenBSD, GNU, ...)

Reply:
OK. I hope that code is not broken but we'll see when experimenting with ports.